### PR TITLE
feat(MeetingsJsonAdapter): implement getMeeting

### PR DESCRIPTION
**Pull request by adamweeks**
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-100969

### CODE REVIEW

File: src/adapters/MeetingsJSONAdapter.test.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import meetings from './../data/meetings';
+import MeetingsJSONAdapter from './MeetingsJSONAdapter';
+
+describe('Meetings JSON Adapter Interface', () => {
```
Comment (lalli-flores) [inline src/adapters/MeetingsJSONAdapter.test.js]:
```suggestion
describe('Meetings JSON Adapter', () => {
```

File: src/adapters/MeetingsJSONAdapter.test.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    });
+  });
+
+  test('getMeeting() throws a proper error message', (done) => {
+    const wrongMeetingID = 'wrongMeetingID';
+
+    meetingsJSONAdapter.getMeeting(wrongMeetingID).subscribe(
+      () => {},
+      (error) => {
+        expect(error.message).toBe(`Could not find meeting with ID "${wrongMeetingID}"`);
```
Comment (lalli-flores) [inline src/adapters/MeetingsJSONAdapter.test.js]: Should the expect be hardcoded?
e.g.
```suggestion
        expect(error.message).toBe('Could not find meeting with ID "wrongMeetingID"');
```

Comment (adamweeks) [reply src/adapters/MeetingsJSONAdapter.test.js]: No, it doesn't need to be hard coded. This matches with the consistency for all other JSON tests. Also gives us the ability to change the actual wrong meeting ID at some point if needed.

**Pull request merged**